pages/sign-up.js

```diff
@@ -48,52 +51,59 @@ const SignUp = () => {
                         </p>
                       </div>
                       <HookForm onSubmit={onSubmit} schema={SignUpSchema}>
-                        <Input label="Email address" name="email" type="email" />
+                        <Input text="Email address" name="email" type="email" />
```

Comment:
Sorry about this, lets revert this back to `label` since `label` makes more sense here than `text`. Please fix for other components too.